### ISSUE TITLE
Close button appears on pinned tabs, making it very difficult to select

### ISSUE THREAD

**Issue by podiki** [akshat46/FlyingFox#70]
**Describe the bug**
Pinned tabs have a close button appear when hovering over them, making it difficult to select the tab with mouse rather than close it. This seems to happen somewhat randomly: right now all 5 pinned tabs in one window have this, while another window with 4 pinned tabs does not have the problem. Unpinning and repinning will sometimes fix it, though tried again now and only fixed it for a second (maybe until changing tab focus).

**To Reproduce**
1. Pin a tab
2. Try to select it with mouse
3. Instead have the close tab icon appear, obscuring nearly the whole tab

**Screenshots**
![ffbug](https://user-images.githubusercontent.com/7770736/115423740-13a65400-a1cc-11eb-89d5-ab23d0c436ce.png)
(A little tricky to tell, but mouse is hovering over the fifth pinned tab)

 **Firefox Version:** 88.0 (and happened on at least 87.0 too)

**Comment by akshat46** [comment]
I know its been a while but I've been using tabline for last 2 days in hopes of running into this issue but haven't yet. If you are still seeing this bug it would really help if there was a certain way to recreate it.

**Comment by podiki** [comment]
It is frustratingly random to me, really not sure why. Seems to be happening less? But maybe because I've been restarting Firefox more?

Anything you might suggest I do or can look at to help debug? I'll see if there any reproducible steps, though might be more what makes it go away once it happens (some combo of opening/closing tabs maybe).

**Comment by podiki** [comment]
Of course, just after I write that comment, the bug happens and also goes away. I tried opening and closing new and pinned tabs to no effect. Then later, after I closed tabs that were already open, and the closed buttons on the pin tabs no longer appears. No idea if that actions were related, but that's what I have so far.

Any clues in that? I'll try to keep doing the same thing when it happens to see if the "fix" is reproducible.

**Comment by podiki** [comment]
One more clue: might have to do with size of tabs/number? Right now with 6 pinned tabs and 8 regular tabs, no problem. Opening one more regular tab causes the problem. Resizing the window to wider makes it go away, as does closing the new tab. (I'm on Linux with XMonad, Xorg, and on a 4k monitor, if that helps. Probably rather in the minority in pixels/size here.)
